src/traces/pie/index.js

```diff
@@ -23,6 +23,7 @@ Pie.moduleType = 'trace';
 Pie.name = 'pie';
 Pie.basePlotModule = require('../../plots/cartesian');
 Pie.categories = ['pie', 'showLegend'];
+Pie.layoutCategories = ['pie'];
```

Comment:
**IMPORTANT** pie plots now have `_has('cartesian')` **AND** `_has('pie')` return true.